crates/project_model/src/cargo_workspace.rs

```diff
@@ -80,19 +80,35 @@ pub type Package = Idx<PackageData>;
 
 pub type Target = Idx<TargetData>;
 
+/// Information associated with a cargo crate
 #[derive(Debug, Clone, Eq, PartialEq)]
 pub struct PackageData {
+    /// Version given in the `Cargo.toml`
     pub version: String,
+    /// Name as given in the `Cargo.toml`
     pub name: String,
+    /// Path containing the `Cargo.toml`
     pub manifest: AbsPathBuf,
+    /// Targets provided by the crate (lib, bin, example, test, ...)
     pub targets: Vec<Target>,
+    /// Is this package a member of the current workspace
     pub is_member: bool,
+    /// List of packages this package depends on
     pub dependencies: Vec<PackageDependency>,
+    /// Default Rust edition for this package
     pub edition: Edition,
+    /// List of features to activate
```

Comment:
```suggestion
    /// List of available features
```

Reply:
From what I could gather, this field is taken from the [Resolve](https://docs.rs/cargo_metadata/0.12.0/cargo_metadata/struct.Resolve.html) structure of `cargo metadata`, which contains [enabled](https://docs.rs/cargo_metadata/0.12.0/cargo_metadata/struct.Node.html#structfield.features) features.

Testing by hand also confirmed this